modules/ROOT/pages/configuring-sso.adoc

```diff
@@ -0,0 +1,440 @@
+|*SAML JIT Handler* |*AEHPortalRegistrationHandler*
+|*Executed Handler As* |Select a system administrator.
+|===
++
+image::aeh-sso-saml-single-sign-on-settings.png[SAML Single Sign-On Settings example]
+====
+
+.Microsoft Azure SAML
+[%collapsible]
+====
+
+Create a new application for the API Experience Hub portal in the identity provider using Microsoft Azure. 
+
+. In Azure, create a SAML Enterprise application:
+.. Go to the Azure administration portal or enter `https://portal.azure.com` in a browser. 
+.. Go to Microsoft Entra Id (Active Directory service). 
+.. Click *+ Add* and select *Enterprise application*.
+.. In the *Browse Microsoft Entra Gallary* select the *Salesforce* application.
+.. Enter a name such as *AEH Portal* and click *Create*.
+. Configure the single sign-on settings for the application:
+.. From *Manage*, select *Single sign-on* and select *SAML*. 
+.. From *Basic SAML Configuration*, click *Edit*.
+.. Complete the following required fields:
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Identifier (Entity ID)|https://<YOUR_DOMAIN>.my.site.com/aeh
+|Reply URL (Assertion Consumer Service URL)| https://<YOUR_DOMAIN>.my.site.com/aeh/login
+|Sign on URL |https://<YOUR_DOMAIN>.my.site.com/aeh/login
+|Logout Url (Optional)|https://<YOUR_DOMAIN>.my.site.com/aeh/services/auth/sp/saml2/logout
+|===
+.. Click *Save*.
+. Configure the required first name, last name, email, and username claims that the application sends:
+.. From *Attributes & Claims*, click *Edit*.
+.. Click *Add a group claim*, select *All Groups* or any other option except *None*, and click *Save*. 
++
+At least one group must be associated to perform a group mapping in a later step.
+.. Add a new claim for each required claim by clicking *+ Add new claim* and completing the following fields for each claim:
++
+* First Name
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Name|firstName
+|Source attribute|user.givenname
+|===
++ 
+* Last Name
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Name|lastName
+|Source attribute|user.surname
+|===
++
+* Email
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Name|email
+|Source attribute|user.email
+|===
++
+* Username
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Name|email
+|Source attribute|user.principalname
+|===
+.. Click *Save*.
+
+*Map the Application to the Identity Provider for Anypoint Platform*
+
+If this single sign-on is already configured in Anypoint, map the application to the identity provider for Anypoint Platform.
+When the single sign-on is already configured in Anypoint, you must map the identity provider ID with the anypoint_idp_id by adding and configuring a new custom attribute called anypoint_idp_id.
+
+If usernames for the portal and Anypoint Platform don’t match, add another custom attribute called anypoint_username to force an update for existing identity provider users to match the identity in the portal to the users in Anypoint Platform.
```

Comment:
thought: The anypoint_username is used to explicitly tell which is going to be the Anypoint Username.
If this field is set after the user was created, it's not going to update the Anypoint username.